gqcpy/src/QCModel/HF/UHF_bindings.cpp

```diff
@@ -34,19 +34,17 @@ void bindQCModelUHF(py::module& module) {
 
         .def(
             "calculateOrthonormalBasis1DM",
-            [](const GQCP::QCModel::UHF<double>& uhf_parameters, const GQCP::Spin sigma) {
-                return uhf_parameters.calculateOrthonormalBasis1DM(sigma);
+            [](const GQCP::QCModel::UHF<double>& uhf_parameters) {
+                return uhf_parameters.calculateOrthonormalBasis1DM();
             },
-            py::arg("sigma"),
-            "Return the sigma-spin UHF 1-DM expressed in an orthonormal sigma spin-orbital basis for these UHF model parameters.")
+            "Return the spin resolved UHF 1-DM expressed in an orthonormal sigma spin-orbital basis for these UHF model parameters.")
 
         .def(
             "calculateScalarBasis1DM",
-            [](const GQCP::QCModel::UHF<double>& uhf_parameters, const GQCP::Spin sigma) {
-                return uhf_parameters.calculateScalarBasis1DM(sigma);
+            [](const GQCP::QCModel::UHF<double>& uhf_parameters) {
+                return uhf_parameters.calculateScalarBasis1DM();
             },
-            py::arg("sigma"),
-            "Return the sigma-spin UHF 1-DM expressed in the underlying scalar basis for these UHF model parameters.")
+            "Return the spin resolved UHF 1-DM expressed in the underlying scalar basis for these UHF model parameters.")
```

Comment:
Since the Python bindings don't have the density matrices yet, any Python calls to these APIs will result in an error. Therefore, I'd like to propose for you to work on #677 after this one.

Reply:
I will implement these bindings (from issue #677) in this PR, in order to make the UHF bindings work.